src/env/dev/terraform.tfvars

```diff
@@ -23,6 +23,7 @@ cidr_subnet_mock_psp = ["10.1.240.8/29"]
 
 cidr_subnet_checkout_be = ["10.1.240.17/29"]
 cidr_subnet_buyerbanks  = ["10.1.240.24/29"]
+cidr_subnet_proxy       = ["10.1.240.32/29"]
```

Comment:
`proxy` too generic, better `pagopaproxy` (using the same naming convention on the resources present in[ IO subscription](https://github.com/pagopa/io-infrastructure-live-new/blob/43d89ce34ecef945dadefdb1dc6dc5473a1bd20a/prod/westeurope/external/pagopaproxyprod/app_service/terragrunt.hcl#L52) ). 
p.s it is also valid for all resources/variables of this PR